gamemodes/terrortown/entities/weapons/weapon_ttt_health_station.lua

```diff
@@ -52,11 +52,7 @@ SWEP.builtin = true
 SWEP.AllowDrop = false
 SWEP.NoSights = true
 
----
--- @ignore
-function SWEP:OnDrop()
-	self:Remove()
-end
+SWEP.drawColor = Color(180, 180, 250, 255)
```

Comment:
Why did you change this? Was this broken before?

It would help if you can outline the reasoning behind your changes in a quick summary in PR descriptions in the future :)

Reply:
the drawcolor was not stored anywhere, and it needs to be somewhere in order for the dropped worldmodel to be tinted, because only the entity is tinted

Reply:
(b/c doing it in DrawWorldModel would create garbage every frame)